app/lib/aggregates.rb

```diff
@@ -0,0 +1,78 @@
+module Aggregates
+  class Calculations
+
+    # total number of animals in the program, by spawning date and holding facility
+    # spawndate acts as the cohort designator here
+    def self.total_animals_by_spawndate_and_facility(spawning_date, facility)
+      PopulationEstimate.not_raw
+        .where(facility: facility, spawning_date: spawning_date)
+        .pluck(:abundance)
```

Comment:
@ThunderHeavyIndustries Can you use .sum(:abundance) here instead of pluck, map and reduce?

Reply:
string field